### ISSUE TITLE
Decorators don't apply on Default Search page

### ISSUE THREAD

**Issue by Bendolfc** [Graylog2/graylog2-server#21165]
When a decorator is created for the Default search, it doesn't seem to take affect.

## Expected Behavior
Decorators created for default search should apply on /search page

## Current Behavior
Decorator is not applied, the field is not created or modified.

## Possible Solution
N/A

## Steps to Reproduce (for bugs)

1. Create decorator for Default Search
2. Perform search on /search page

You will observe no decorator applied

## Context
I am finally getting rid of an old Graylog 2 install where we make extensive use of decorators on the search page. I need to replicate those decorators but the only way I can get decorators to apply currently is to go in and search in a stream rather than on /search. 

## Your Environment

* Graylog Version: 6.1.4

![Image](https://github.com/user-attachments/assets/04254cd0-3d2d-46cc-a59e-55d6954e52dc)

**Comment by Bendolfc** [comment]
Yes, its entirely possible DEFAULT_SEARCH on the UI doesn't mean what I assumed it does.

Thinking about it logically though, even if that was the intent of Default Search, it probably doesn't make much sense as applying decorators to some types of logs as for some they would make no sense. It would make more sense to just apply the relevant streams decorator for each log entry that is returned.

For now, I'll educate my users to navigate to the relevant stream search to get the decorated results.

**Comment by drewmiranda-gl** [comment]
For what its worth i went back testing each Graylog version and the most recent version I could get this to work on:
<img width="522" alt="Image" src="https://github.com/user-attachments/assets/458822e8-5506-4cf6-bc22-ba4e0425004e" />

Is Graylog 4.3

I was able to replicate this bug with 5.0, 5.1, 5.2, 6.0, 6.1
